pkg/option/config.go

```diff
@@ -2536,6 +2537,17 @@ func ReplaceDeprecatedFields(m map[string]interface{}) {
 	}
 }
 
+func CheckExistingFlagCLI(val string) bool {
```

Comment:
Maybe change the name as follows?
```suggestion
func IsKnownCLIFlag(val string) bool {
```